### PR TITLE
Created basic json config file and functions for it

### PR DESCRIPTION
**Pull request by ImTheTom**
Created a basic Json file and functions that remembers the settings when you last ran it. At the moment only the upper limit, lower limit, color name, hex color works, wanted to see if this is what you meant in the future milestones around the adding a config file.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        update_json_file_setting("Color", color)
+        if(hex_value is not None):
+            update_json_file_setting("Hex Color", hex_value)
+    except IOError:
+        print("Could not update Json file")
+
+
+def get_hex_color():
+    try:
+        return get_json_file_setting("Hex Color")
```
Comment (djmgit) [inline config.py]: For json keys, lets follow camel case, also putting space in key names is not a good practice. So please change it to HexColor.

File: config.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    try:
+        return get_json_file_setting("Upper Limit")
+    except IOError:
+        print("Could not retrieve information from Json file")
+        return 100
+
+
+def set_upper_limit(value):
+    try:
+        update_json_file_setting("Upper Limit", value)
```
Comment (djmgit) [inline config.py]: same here

File: config.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    try:
+        return get_json_file_setting("Lower Limit")
+    except IOError:
+        print("Could not retrieve information from Json file")
+        return 0
+
+
+def set_lower_limit(value):
+    try:
+        update_json_file_setting("Lower Limit", value)
```
Comment (djmgit) [inline config.py]: same here

**Review by djmgit — CHANGES_REQUESTED** [review]
Nice work. I think this is a nice feature and great UX improvement. 
There is one issue, you have saved the app the theme however you are not applying it, when the app is launched. Try switching to dark theme then restart the app, the app starts with light theme applied. Please check this out. Also I have left some comments. Please have a look.

Nice work and welcome on board :) :+1:

**Comment by djmgit** [comment]
@Aniq55 Please have a look at this PR, I feel this is a nice feature and @ImTheTom has done a great job. What are your views?

**Comment by Aniq55** [comment]
@djmgit yes, this PR was much needed. More features can be added to this, after the merge, I'll open the requisite issues then.

**Comment by djmgit** [comment]
@Aniq55 great :+1:

**Pull request closed without merging**